playwright/src/test/java/com/microsoft/playwright/TestBrowserContextBasic.java

```diff
@@ -92,11 +97,11 @@ void shouldIsolateLocalStorageAndCookies() {
     context1.close();
     context2.close();
 
-    assertEquals(1, browser.contexts().size());
+    assertEquals(0, browser.contexts().size());
```

Comment:
these assertions needed to be updated because the TestBase used to create a context but fixtures don't create one unless you ask for it.